libsolidity/codegen/MultiUseYulFunctionCollector.cpp

```diff
@@ -43,6 +46,7 @@ string MultiUseYulFunctionCollector::createFunction(string const& _name, functio
 {
 	if (!m_requestedFunctions.count(_name))
 	{
+		m_requestedFunctions[_name] = "STUB";
```

Comment:
```suggestion
		m_requestedFunctions[_name] = "<<STUB<<";
```
Just to make sure we really get a parser error if this ends up somewhere